<launch>
    <!-- Sound play node -->
    <node name="soundplay_node" pkg="sound_play" type="soundplay_node.py"/>

    <!-- Music play file -->
     <node name="music_play_node" pkg="sound" type="music_play_node" output="screen">
        <param name="music_file_path" value="/home/deguchi-a/demo/src/sound/src/bgm1.mp3"/> 
    </node>

    <!-- Effect play file -->
    <node name="effect_play_publish_node" pkg="sound" type="effect_play_publish_node" output="screen"/>

    <!-- Effect play file -->
    <node name="effect_play_subscribe_node" pkg="sound" type="effect_play_subscribe_node" output="screen"/>
</launch>
